src/main/java/seedu/address/logic/commands/CountryNoteCommand.java

```diff
@@ -13,8 +15,14 @@
 public class CountryNoteCommand extends Command {
 
     public static final String COMMAND_WORD = "country note";
-    private static final String MESSAGE_DUPLICATE_COUNTRY_NOTE = "The country note already exists";
-    private static final String MESSAGE_SUCCESS = "Successfully added country note";
+    private static final String MESSAGE_DUPLICATE_COUNTRY_NOTE = "This country note already exists in TBM";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Adds a note that is associated with the user input country.\n"
+            + "Parameters: "
+            + PREFIX_COUNTRY + "COUNTRY_CODE "
+            + PREFIX_NOTE + "NOTE_STRING\n"
+            + "Example: " + COMMAND_WORD + " c/SG nt/likes laksa";
```

Comment:
Would be better to move this above MESSAGE_DUPLICATE_COUNTRY_NOTE so that the public static final variables are tgt.

Reply:
Done